h/views/api/bulk/stats.py

```diff
@@ -47,3 +53,35 @@ def assignment(request):
         status=200,
         content_type="application/x-ndjson",
     )
+
+
+@api_config(
+    versions=["v1", "v2"],
+    route_name="api.bulk.stats.course",
+    request_method="POST",
```

Comment:
Considering this endpoint has no side effects, should it be using `GET` and receive the filtering options via query params?

I know that's sometimes a bit limiting and a JSON body is more flexible, so we don't have to be super REST compliant if being pragmatic makes more sense here.

If only the [QUERY](https://www.ietf.org/archive/id/draft-ietf-httpbis-safe-method-w-body-02.html) method was already a thing 😅

Reply:
Yeah I reckon these make sense as POST for now event if it's very un-RESTFUL.

At least this is an internal API, we should be more careful in public APIs.